test/specs/cli.js

```diff
@@ -134,6 +134,15 @@ describe('cli', () => {
       output.code.should.equal(2);
     });
 
+    it('uses --negate to allows for "not" conditions ', () => {
+      const woNegate = cli('test', '-d', 'fakeDirName');
```

Comment:
nit: please spell out "without" and "with" in the variable names. It just helps with readability a bit.

Reply:
Will do !